introducing-signal-store.md

```diff
@@ -0,0 +1,701 @@
+    ...state,
+    list: state.list.filter((item) => item.id !== payload.id),
+  })),
+);
+```
+_FYI_ the powerful [ts-action](https://www.npmjs.com/package/ts-action) library is used to create reducers with less boilerplate.
+
+#### Effects
+
+You create an effect like this: 
+- Listen to a specific action
+- Run an (in most cases) asynchronous task
+- Return a new action, when the task succeeded/failed
+
+```ts
+import { inject, Injectable } from '@angular/core';
+import {
+  Actions,
+  createRxEffect,
+  mapResponse,
+} from '@mini-rx/signal-store';
+import { ofType } from 'ts-action-operators';
+import { ProductApiService } from '../product-api.service';
+import { mergeMap } from 'rxjs';
+import {
+  loadProducts,
+  loadProductsError,
+  loadProductsSuccess,
+} from './product.actions';
+
+@Injectable()
+export class ProductEffects {
+  actions$ = inject(Actions);
+  todosApi = inject(ProductApiService);
+
+  loadTodos$ = createRxEffect(
+    this.actions$.pipe(
+      ofType(loadProducts),
+      mergeMap(() =>
+        this.todosApi.getTodos().pipe(
+          mapResponse(
+            (res) => loadProductsSuccess(res),
+            (err) => loadProductsError,
+          ),
+        ),
+      ),
+    ),
+  );
+}
+```
+
+#### Register reducers and effects
+
+You can register reducers and effects with modern Angular standalone APIs (`provideStore`, `provideEffects`) when initializing the app:
+```ts
+import { ApplicationConfig } from '@angular/core';
+import { routes } from './app.routes';
+import {
+  ImmutableStateExtension,
+  provideEffects,
+  provideStore,
+  ReduxDevtoolsExtension,
+} from '@mini-rx/signal-store';
+import { productReducer } from './products/state/product.reducer';
+import { ProductEffects } from './products/state/product.effects';
+
+export const appConfig: ApplicationConfig = {
+  providers: [
+    provideStore({
+      reducers: {
+        product: productReducer,
+      },
+      extensions: [
+        new ReduxDevtoolsExtension({ name: 'Signal Store Demo' }),
+        new ImmutableStateExtension(),
+      ],
+    }),
+    provideEffects(ProductEffects),
+  ],
+};
+```
+
+#### Register reducers and effects via a lazy loaded component
+
+It is possible to register reducers together with a lazy loaded component (see `provideFeature`).
+For registering effects you can use `provideEffects`.
+
+```ts
+import { Routes } from '@angular/router';
+import { ProductShellComponent } from './products-shell/product-shell.component';
+import { productReducer } from './state/product.reducer';
+import {
+  provideEffects,
+  provideFeature,
+} from '@mini-rx/signal-store';
+import { ProductEffects } from './state/product.effects';
+
+export const productRoutes: Routes = [
+  {
+    path: '',
+    component: ProductShellComponent,
+    // Lazy load the products state
+    providers: [
+      provideFeature('products', productReducer),
+      provideEffects(ProductEffects),
+    ],
+  },
+];
+```
+
+#### Component usage
```

Comment:
it was not clear for me in the beginning - is that "component store" or "minirx redux store used in components".
After reading the thing I know it's the 2nd option. However, consider making the subtitle more clear ;)

Reply:
indeed not clear